docs/DeveloperGuide.md

```diff
@@ -502,148 +502,174 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use Cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
-
-**Use case: Delete a company**
+<div markdown="block" class="alert alert-info">
+**:information_source: Note:**<br>
+For all use cases below, the **System** is `LinkMeIn` and the **Actor** is the `user`, unless specified otherwise.
+</div>
 
-**MSS**
 
-1.  User requests to list companies
-2.  AddressBook shows a list of companies
-3.  User requests to delete a specific company in the list
-4.  AddressBook deletes the company
+**Use Case: UC01 - List all Companies**
 
-    Use case ends.
+**MSS** <br>
+1. User requests to list all companies. 
+2. LinkMeIn displays the full list of companies. <br>
+Use case ends.
 
-**Extensions**
+**Extensions** <br>
+* 1a. LinkMeIn detects an invalid command format error in the input.
+  * 1a1. LinkMeIn displays an error message. 
+  * 1a2. User enters a new command. <br>
+  Use case resumes from Step 1.
```

Comment:
Should be resume from step 2? Since step 1a2 the user already entered a command.
Same for the other use cases too

Reply:
For this, but if we go to step 2, then it's assuming that the new command is correct? which might not be the case?

Reply:
how about "Steps 1a1-1a2 are repeated until user enters a valid command format" "Use case resumes from step 2"